フラグ"PREVENT_SUDDEN_MAGIC"の廃止

dis- opened the pull request:
resolve #4241
召喚即魔法行動を抑制するフラグだが、終盤の強力なモンスターほぼ全てに配られているためフラグとして存在する意味がない。
新規モンスター実装の際つけ忘れるなど良いことがないため、フラグを廃止して全てのモンスターに対して召喚即魔法を抑制する。(悪夢モード時を除く)

Hourier reviewed (dismissed):
対応お疲れ様です
修正内容及び起動確認しました
特に問題なさそうです、Approveします

dis- committed:
[Fix] PREVENT_SUDDEN_MAGICフラグの廃止

resolve #4241
召喚即魔法行動を抑制するフラグだが、終盤の強力なモンスターほぼ全てに配られているためフラグとして存在する意味がない。
新規モンスター実装の際つけ忘れるなど良いことがないため、フラグを廃止して全てのモンスターに対して召喚即魔法を抑制する。(悪夢モード時を除く)

dis- commented:
改行コードの問題あったので、#4250 を前提として再構成しました。
再度確認お願いします。

Hourier reviewed (approved):
対応お疲れ様です
特に問題ありません
Approveします

dis- commented:
確認ありがとうございました
